archive.mk

```diff
@@ -39,7 +39,7 @@ archive.json: fetch-archive $(drafts_source)
 	old_archive=$$(mktemp /tmp/archive-old.XXXXXX); \
 	trap 'rm -f $$old_archive' EXIT; \
 	git show $(ARCHIVE_BRANCH):$@ > $$old_archive || true; \
-	$(trace) archive -s archive-repo $(python) -m archive-repo archive $(GITHUB_REPO_FULL) $(GITHUB_API_TOKEN) $@ --reference $$old_archive;
+	PYTHONPATH=$(VENV) $(trace) archive -s archive-repo $(VENV)/python -m archive-repo archive $(GITHUB_REPO_FULL) $(GITHUB_API_TOKEN) $@ --reference $$old_archive;
```

Comment:
The best way to manage this is to `export PYTHONPATH`.

Also, I would define `python := $(VENV)/python`

Reply:
defining `python` like that doesn't work in some places, e.g., `build-index.sh`.